scheduler/scheduler-client/src/main/java/org/ow2/proactive/scheduler/task/TaskResultImpl.java

```diff
@@ -71,6 +72,8 @@
     /** The value of the result if no exception occurred */
     private transient Serializable value = null;
 
+    private Map<String, Serializable> JobMap = new ConcurrentHashMap<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "JobMap" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)